src/main/kotlin/racingcar/model/Cars.kt

```diff
@@ -0,0 +1,40 @@
+package racingcar.model
+
+class Cars(private val carList:List<Car>) {
+    val size: Int
+        get() = carList.size
+
+    init{
+        val nameSet = mutableSetOf<String>()
+        for (car in carList) {
+            if (!nameSet.add(car.name)) {
+                throw IllegalArgumentException("중복된 차 이름이 있습니다 : ${car.name}")
+            }
+        }
```

Comment:
중복 체크로 이런 방법도 있습니다. :)
```suggestion
        val carNames = carList.map {it.name}.toSet()
        require(carNames.size == carList.size) { "중복된 차 이름이 있습니다" }
```

Reply:
헐 이런 방법이 있었네요 ㄷㄷ